R/selectionmodels.R

```diff
@@ -195,7 +195,7 @@ SelectionModels <- function(jaspResults, dataset, options, state = NULL) {
 
 .smFillEstimates           <- function(jaspResults, table, fit, options) {
   
-  overtitleCI <- gettext("95% Confidence Interval")
+  overtitleCI <- gettext("95%% Confidence Interval")
```

Comment:
All of these should be `gettextf` otherwise you get a double `%%` in the result

Reply:
Thanks! OK